3p-ecosystem/README.md

```diff
@@ -0,0 +1,101 @@
+
+    ```console
+    Updated Git hooks.
+    Git LFS initialized.
+    ```
+
+3.  Once Git LFS is installed and initialized, you're ready to clone the
+    repository. When cloning completes, Git checks out the `main` branch
+    and downloads the native libraries for you.
+
+    ```console
+    $ git clone https://github.com/google-home/sample-app-for-matter-android.git
+    ```
+
+## Version
+
+Google Home Sample App for Matter follows the [Semantic](http://semver.org/)
+and [Android](https://developer.android.com/studio/publish/versioning) versioning guidelines for
+release cycle transparency and to maintain backwards compatibility.
+
+## Releases
+
+Always use the latest release as shown at
+https://github.com/google-home/sample-app-for-matter-android/releases
+
+## Get started
+
+* Check the presentation "Building Smart Home Apps with the Google Home Mobile SDK"
+    (docs/GoogleHomeMobileSDK.pdf) for an overview of the Sample App and key APIs
+    of the Mobile SDK.
+* To make sure that your device has the latest Matter support, review the
+    [Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services)
+    guide.
+* Build a Matter device with On/Off capabilities. This sample  app works with a virtual device
+    and an ESP32.
+    *   [Build a Matter Virtual Device](https://developers.home.google.com/codelabs/matter-device-virtual)
+        with the `rootnode_dimmablelight_bCwGYSDpoe` app. When you
+        [Create a Matter integration](https://developers.home.google.com/matter/integration/create)
+        in the [Home Developer Console](https://console.home.google.com/projects),
+        use `0xFFF1` as your Vendor ID and `0x8000` as your Product ID.
+    *   [Build an Espressif Device](https://developers.home.google.com/matter/vendors/espressif)
+        with the `all-clusters-app`. When you
+        [Create a Matter integration](https://developers.home.google.com/matter/integration/create)
+        in the [Home Developer Console](https://console.home.google.com/projects),
+        use `0xFFF1` as your Vendor ID and `0x8001` as your Product ID.
+* For an overview of the user interface and features, refer to
+    the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
+* To review code samples and start building, refer to
+    the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
+    Codelab.
+
+## Issues?
+
+If you run into problems with the sample app, please submit an issue on the 
+[Issues page](https://github.com/google-home/sample-app-for-matter-android/issues).
+
+It is always greatly appreciated if you can try to reproduce the issue you encountered 
+from a clean state as described in document
+["Investigating Issues with the Sample App"](https://github.com/google-home/sample-app-for-matter-android/wiki/How-to-investigate-issues-with-the-sample-app-(GHSAFM)).
```

Comment:
Should we update the Wiki to mention 3p-ecosystem etc as well?

app --> apps in the link too